spec/lib/pantry/helpers/category_helper_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe Pantry::Helpers::CategoryHelper, type: :helper do
+
+  describe "#self.categorize" do
```

Comment:
The pattern for identifying class-level methods would just be `.categorize`.  If it was an instance method, it'd be `#categorize`.